examples/volumes/vsphere/README.md

```diff
@@ -353,6 +354,168 @@
       pvpod       1/1     Running   0          48m        
       ```
 
+### Virtual SAN policy support inside Kubernetes
+  __Note: Here you don't need to create vmdk it is created for you.__
```

Comment:
The[ policy settings](https://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vcli.ref.doc%2Fesxcli_vsan.html) can be one or more of the following:
- hostFailuresToTolerate: represents NumberOfFailuresToTolerate
- forceProvisioning: represents if volume must be Force Provisioned 
- stripeWidth:  represents NumberOfDiskObjectsToStripe
- proportionalCapacity: represents ObjectSpaceReservation
- cacheReservation: represents FlashReadCacheReservation